apps/website/package.json

```diff
@@ -28,7 +28,7 @@
     "astro-expressive-code": "^0.33.5",
     "examples": "workspace:*",
     "htm": "^3",
-    "lightningcss": "^1.22.1",
+    "lightningcss": "^1.25.0",
```

Comment:
Unrelated: to confirm, should `lightningcss` be a dev dependency?

Reply:
It doesn't matter, this is not in a published package (e.g. see #1848). And there is no difference for a website built using a framework like astro; everything is a dev dependency.